### ISSUE TITLE
Add juliaup

### ISSUE THREAD

**Issue by heetbeet** [heetbeet/juliawin#66]
For now, just use the Windows build artifacts directly:
https://github.com/JuliaLang/juliaup/actions/runs/3028675600

**Comment by RoyiAvital** [comment]
Any chance to have the new features of `1.8.1` without `juliaup`? Or disable it?

**Comment by heetbeet** [comment]
Sure, are there problems with juliaup?

**Comment by RoyiAvital** [comment]
I prefer not having functionality which is not needed. 
I preferer a direct control over the Julia installation.

**Comment by heetbeet** [comment]
For now, you can delete `packages\julia` and replace it with a Julia version of your choosing (you can actually move `userdata\.julia\juliaup\julia-1.8.1+0.x64` to `packages\julia`). I actually just added `juliaup.exe` and `julialauncher.exe` (renamed to `julia.exe`) to `packages\julia\bin\` in order for a seamless transition.

**Comment by RoyiAvital** [comment]
Any chance to make `juliaup` optional (Or at least a way to opt out) in the next version?

**Comment by USeebi** [comment]
I tried to replace `packages\julia` with julia-1.12.4-win64 and added `juliaup.exe` from juliaup-1.18.9-x86_64-pc-windows-gnu-portable.tar.gz to `packages\julia\bin\`. `juliaup list` now shows all the Julia version beyond 1.8.1 but `juliaup status` shows nothing. It seems that Juliawin only work with Juliaup 1.7.23 which has no update after Julia 1.8.1 while `juliaup self update` gives 

```
Error: get on the return of GetAppAndOptionalStorePackageUpdatesAsync failed.

Caused by:
```

Any workaround?

**Comment by USeebi** [comment]
I probably found the workaround.  I added `set "JULIAUP_DEPOT_PATH=%juliawin_userdata%\.julia"` to activate-juliawin-environment.bat with hints from https://github.com/JuliaLang/juliaup/issues/257#issuecomment-1537235627

Now `juliaup status` shown the installed Julia version and the available latest update. Once I did `juliaup add 1.12.4` for the latest installation and run `julia +1.12.4`, I got the 1.12.4 REPL. However, I also have error on

```
UndefVarError: `Pkg` not defined in `Main`
The binding may be too new: running in world age 38706, while current world is 38730.
Hint: a global variable of this name also exists in Pkg.
```

What did I miss? Which is the offending package?
